src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/Snapshot/AggregateDependenciesSnapshotProvider.cs

```diff
@@ -36,10 +36,7 @@ internal class AggregateDependenciesSnapshotProvider : IAggregateDependenciesSna
         /// <inheritdoc />
         public void RegisterSnapshotProvider(IDependenciesSnapshotProvider snapshotProvider)
         {
-            if (snapshotProvider == null)
-            {
-                return;
-            }
+            Requires.NotNull(snapshotProvider, nameof(snapshotProvider));
```

Comment:
@davidwengier I assume you're referring to this? If so then it's fine, no caller passes `null`.

Reply:
Yep, that's the one.